src/deep-security/lib/CredentialsManager.js

```diff
@@ -0,0 +1,180 @@
+/**
+ * Created by mgoria on 11/6/15.
+ */
+
+'use strict';
+
+import AWS from 'aws-sdk';
+import CognitoSyncManager from 'amazon-cognito-js';
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['CognitoSyncManager' is defined but never used.](https://www.codacy.com/app/MitocGroup/deep-framework/file/1363209931/issues/source?bid=2895475&fileBranchId=2973909#l8)